apps/dashboard/src/pages/integrations/components/integration-card.tsx

```diff
@@ -0,0 +1,117 @@
+import { Badge } from '@/components/primitives/badge';
+import { Button } from '@/components/primitives/button';
+import { RiCheckboxCircleFill, RiGitBranchFill, RiSettings4Line, RiStarSmileLine } from 'react-icons/ri';
+import { ITableIntegration } from '../types';
+import type { IEnvironment, IIntegration, IProviderConfig } from '@novu/shared';
+import { useNavigate } from 'react-router-dom';
+import { ROUTES } from '@/utils/routes';
+import { Tooltip, TooltipContent, TooltipTrigger } from '@/components/primitives/tooltip';
+import { ProviderIcon } from './provider-icon';
+import { cn } from '../../../utils/ui';
+
+interface IntegrationCardProps {
+  integration: IIntegration;
+  provider: IProviderConfig;
+  environment: IEnvironment;
+  onRowClickCallback: (item: { original: ITableIntegration }) => void;
+}
+
+export function IntegrationCard({ integration, provider, environment, onRowClickCallback }: IntegrationCardProps) {
+  const navigate = useNavigate();
+
+  const tableIntegration: ITableIntegration = {
+    integrationId: integration._id ?? '',
+    name: integration.name,
+    identifier: integration.identifier,
+    provider: provider.displayName,
+    channel: integration.channel,
+    environment: environment.name,
+    active: integration.active,
+  };
+
+  const handleConfigureClick = (e: React.MouseEvent) => {
+    if (integration.channel === 'in_app' && !integration.connected) {
+      e.stopPropagation();
+      navigate(ROUTES.INBOX_EMBED);
+    } else {
+      onRowClickCallback({ original: tableIntegration });
+    }
+  };
+
+  const isDemo = isDemoIntegration(provider.id);
+
+  return (
+    <div
+      className={cn(
+        'bg-card shadow-xs group relative flex min-h-[125px] cursor-pointer flex-col gap-2 overflow-hidden rounded-xl border border-neutral-100 p-3 transition-all hover:shadow-lg',
+        !integration.active && 'opacity-75 grayscale'
+      )}
+      onClick={handleConfigureClick}
+      data-test-id={`integration-${integration._id}-row`}
```

Comment:
Novu integrations should have a gradient background:
![Screenshot 2024-12-17 at 17 01 09](https://github.com/user-attachments/assets/beace251-0134-4c36-8a13-7e197fc38ec2)

Reply:
Please also update the border: https://www.figma.com/design/iOS9MnuJjLR7avdSIHemFy?node-id=9685-198980&m=dev#1067690831